source/includes/materials/_01-materials.md.erb

```diff
@@ -0,0 +1,198 @@
+<%
+json = {
+      type: 'git',
+      attributes: {
+        url: 'https://github.com/gocd/api.go.cd',
+        username: 'bob',
+        encrypted_password: 'aSdiFgRRZ6A=',
+        branch: 'master',
+        auto_update: true
```

Comment:
Can we use https://api.gocd.org/19.6.0/#the-pipeline-material-object instead of writing this again?

Reply:
I considered that, but it has the similar issues to re-using the representers in the main codebase. If we change the pipeline config object, we unknowingly end up changing the documentation for Materials as well. 

There's also an additional `fingerprint` attribute on the top level for Materials.

(Btw, this is duplicated in Config Repo documentation as well)